### PR TITLE
17:24	Saegrimr	Then i'm going to start banning nonantags who buy it.

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Saegrimr
del: Removed all shuttles that inconvenience or risk any injury to players, to prevent captains from being banned for purchasing/getting paid to use shuttles that are sub-par, kill, or ruin the fun of the majority of players.
/:cl:

http://pastebin.com/kLhneVL6 full logs for future reference on rulings related to shuttles and captains

chrome_2016-11-18_17-53-25

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
why not make the meme shuttles cost a lot instead of give you points for grieffing the crew?

**Comment by Iamgoofball** [comment]
you'll still get banned for griefing the crew

and they gave credits so that you'd have an incentive to actually get them

**Comment by Kraseo** [comment]
Repeat after me:

# **H U G B O X**

**Comment by Iamgoofball** [comment]
@Kraseo it's bannable so

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -178,20 +138,6 @@
...
-	description = "The research team based on this vessel went missing one \
-	day, and no amount of investigation could discover what happened to \
-	them. The only occupants were a number of dead rodents, who appeared to \
-	have clawed each other to death. Needless to say, no engineering team \
-	wanted to go  near the thing, and it's only being used as an Emergency \
-	Escape Shuttle because there is literally nothing else available."
-	admin_notes = "If the crew can solve the puzzle, they will wake the \
-	wabbajack statue. It will likely not end well. There's a reason it's \
-	boarded up. Maybe they should have just left it alone."
-	credit_cost = 15000
```
Comment (Iamgoofball) [inline code/datums/shuttles.dm]: dangerous and can get players easily griefed

File: code/datums/shuttles.dm [hunk 2]
```diff
@@ -159,17 +130,6 @@
...
-/datum/map_template/shuttle/emergency/imfedupwiththisworld
-	suffix = "imfedupwiththisworld"
-	name = "Oh, Hi Daniel"
-	description = "How was space work today? \
-		Oh, pretty good. We got a new space station and the company will make a lot of money. \
-		What space station? \
-		I cannot tell you; it's space confidential. \
-		Aw, come space on. Why not? \
-		No, I can't. Anyway, how is your space roleplay life?"
-	credit_cost = -5000
```
Comment (Iamgoofball) [inline code/datums/shuttles.dm]: small amount of space and spawns with a gun you can grief players with which is what usually happens

File: code/datums/shuttles.dm [hunk 3]
```diff
@@ -99,21 +85,6 @@
...
-		they probably weren't expecting you to need evac for a while. \
-		Probably best if you don't rifle around in whatever equipment they \
-		were transporting. I hope you're friendly with your coworkers, \
-		because there is very little space in this thing.\n\
-		\n\
-		Contains contraband armory guns, maintenance loot, and abandoned \
-		crates!"
-	admin_notes = "Due to origin as a solo piloted secure vessel, has an \
-		active GPS onboard labeled STV5."
-	credit_cost = -7500
```
Comment (Iamgoofball) [inline code/datums/shuttles.dm]: spawns with loads of guns, cramped space, lots of grief potential

File: code/datums/shuttles.dm [hunk 4]
```diff
@@ -57,20 +57,6 @@
 		medical facilities."
 	credit_cost = 5000
 
-/datum/map_template/shuttle/emergency/meteor
-	suffix = "meteor"
-	name = "An Asteroid With Engines Strapped To It"
-	description = "A hollowed out asteroid with engines strapped to it. Due to its size and difficulty in steering it, this shuttle may damage the docking area."
-	admin_notes = "This shuttle will likely crush escape, killing anyone there."
-	credit_cost = -5000
```
Comment (Iamgoofball) [inline code/datums/shuttles.dm]: press button receive ban for non-antag bombing all of escape

**Comment by ExcessiveUseOfCobblestone** [comment]
> and they gave credits so that you'd have an incentive to actually get them

That's the problem, they give you points to antagonize the crew [as nonantag] FNR.

**Comment by Iamgoofball** [comment]
removing the incentive isn't going to stop them from griefing the crew

**Comment by Kraseo** [comment]
If people can't get out of the way with CLEARLY MARKED PLACES THE SHUTTLE IS GOING TO CRASH INTO they don't deserve living tbh.

**Comment by Saegrimr** [comment]
Good job only posting part of the IRC logs you liked you snarky cunt.

http://pastebin.com/kLhneVL6

**Comment by Iamgoofball** [comment]
i was too lazy to screencap it all, just grabbed the important bit

will add to OP, thanks bae

**Comment by Saegrimr** [comment]
Good. If you're gonna try and make me look bad, use the whole thing so I don't feel like I wasted an entire 20 minutes of arguing for nothing.

**Comment by Iamgoofball** [comment]
👌

**Comment by Iamgoofball** [comment]
don't see it as a waste btw, i'm saving you time from having to deal with appeals over shuttle purchases by doing that

**Comment by KorPhaeron** [comment]
>they give an incentive to do something for no reason

???

**Comment by Iamgoofball** [comment]
an extra 5000 cargo points can buy a lot of meme shit kor

**Comment by B00tz** [comment]
Why not have it so when a dangerous shuttle is bought it lists the dangers of it so people know to stand farther and such.

**Comment by ExcessiveUseOfCobblestone** [comment]
yeah sorry im dumb.

They give an incentive to grief the crew as [usually] a nonantag. Just that

File: code/datums/shuttles.dm [hunk 5]
```diff
@@ -83,37 +69,6 @@
...
-		Well then order the Snappop(tm) today! We've got fun activities for \
-		everyone, an all access cockpit, and no boring security brig! Boo! \
-		Play dress up with your friends! Collect all the bedsheets before \
-		your neighbour does! Check if the AI is watching you with our patent \
-		pending \"Peeping Tom AI Multitool Detector\" or PEEEEEETUR for \
-		short. Have a fun ride!"
-	admin_notes = "Brig is replaced by anchored greentext book surrounded by \
-		lavaland chasms, stationside door has been removed to prevent \
-		accidental dropping. No brig."
-	credit_cost = 8000
```
Comment (Iamgoofball) [inline code/datums/shuttles.dm]: easy to permakill people via shuttle grief by opening the all access door and throwing people into chasm tiles

**Comment by KorPhaeron** [comment]
Should buying destructive shuttles (like the giant asteroid) result in a ban for the captain?
Yes, it kills too many people: 2
No, it's funny and people can move out of the way: 37
I don't know what you're talking about or care very much either: 4
Did not vote: 31

**Comment by Saegrimr** [comment]
>Why not have it so when a dangerous shuttle is bought it lists the dangers of it so people know to stand farther and such.

This was offered as a suggestion, which I replied that the

>CLEARLY MARKED PLACES THE SHUTTLE IS GOING TO CRASH INTO they don't deserve living tbh.

Aren't noticable on the station tiles. If the area is packed, nobody is going to be able to push eachother out of the way in time.

It's good that goof is being a bigger memelord than I am just to get his own PR shut down to try and prove a point.

**Comment by ExcessiveUseOfCobblestone** [comment]
Right so if we don't want to ban them [which I have no problem with] then make it more difficult to obtain the "lol kill the crew" shuttles by making them cost more [and certainly not give you points].

**Comment by Niknakflak** [comment]
Just close this salt fueled bullshit

**Comment by Incomptinence** [comment]
@Saegrimr 

Seems like everything was find before your back seat headmin/headcodering.

At most requiring a new sprite for the indicators.

@Niknakflak 

Typical niknak

**Comment by Saegrimr** [comment]
>At most requiring a new sprite for the indicators.

Taken from the round just minutes ago on Sybil.
Indicators as they arrive, barely noticable.
http://i.imgur.com/WYVRJ9O.png

Max brightness, this is however too late as the shuttle is arriving right now.
http://i.imgur.com/d3fHa1R.png

Destroyed everything up to the science entrance doors, R&D just plain gone.
http://i.imgur.com/MTYeYNC.png

Not even electrical maint is safe.
http://i.imgur.com/pfa9Tsl.png

[19:47:37] <Saegrimr> You say that like we haven't banned people for bombing the shuttle or just shuttle grief in general.

Every round is this shitty meme of a shuttle, 5000 credits is practically fucking nothing to justify it with.
To quote the fucking MotD.

" Try not to run this into the ground."

**Comment by Bawhoppen** [comment]
goof being trash again

**Comment by Kraseo** [comment]
"barely noticeable" if you're as blind as a bat maybe

**Comment by Niknakflak** [comment]
@Incomptinence 
It's one thing to have a debate about reverting the asteroid shuttle.  It's another for goofball to go in and remove a shit ton of other shuttles that don't really have any problems because he makes up some bullshit "grief potential".

Take your shitty bias elsewhere.

**Comment by Saegrimr** [comment]
@Kraseo 

Nice contrast

Seriously?

**Comment by KorPhaeron** [comment]
seemsfine

??

**Comment by Saegrimr** [comment]
@KorPhaeron 
>Max brightness, this is however too late as the shuttle is arriving right now.

Yes thank you for showcasing the next screenshot I had.

**Comment by Saegrimr** [comment]
While we're also on the topic of pasting log snips and goofball not understanding the difference between gibbing an entire area full of people and having to sit on someone's lap.

(iamgoofball) and would you apply this "fun ruining" issue to any of the other shuttles that inconvenience crew
(Saegrimr) Wasn't there one that was literally just plating and no walls and airless? That's probably also a bad idea.
(iamgoofball) no but there's stuff like small cramped shuttles, a shuttle you can only enter if you have 500 bucks on you, etc.
(Saegrimr) That one is hilarious but it still has a poverty section for people to get on.
(iamgoofball) yes but it's small and cramped and most the station can't get on it
(iamgoofball) therefore, it's ruining the fun of the people who can't get on
(Saegrimr) b-but the captain's fun
(iamgoofball) there you go
(Saegrimr) Do you not see how gibbing the escape hallway and everything around it is worse than cramped spaced?
(Saegrimr) Besides i've seen 20 people fit on a pod with laying down. Up until someone opened the door anyway.
(iamgoofball) i'm removing the asteroid shuttle now
(iamgoofball) i just don't think dying in a 30 minute round based game is a bad thing when it gives you access to the grief lottery on steroids within the next 5-10 minutes
(Saegrimr) You say that like we haven't banned people for bombing the shuttle or just shuttle grief in general.
(iamgoofball) i don't think it matters all that much either but my opinion doesn't matter

**Comment by KorPhaeron** [comment]
I've personally never had any trouble moving out of the way of the ripples, even for the asteroid shuttle.

This is going away from the broader argument of whether the captain should have round altering powers/authority though.

**Comment by Kraseo** [comment]
@Saegrimr I mean fuck there's a slight flash before the circles, and even then it gradually gets brighter so I dunno how you can miss it.

**Comment by KorPhaeron** [comment]
Anyway I think it's a kneejerk reaction to ban people for overusing it when it's both less than a week old and overwhelmingly supported by players right now. There are very few things in the game less controversial at this time.

Rules are to make sure people have a fun time, and right now people are still having a fun time with it.

I'll fuck off now and let the headmins decide

**Comment by Saegrimr** [comment]
@KorPhaeron 
Shame about the whole "Try not to run this into the ground." thing when its bought immediately every round completely irrelevant of cargo's status.

@Kraseo 
Why should people who don't get to see the message about HUGE ASTEROID at roundstart have to expect the escape shuttle to show up on top of them and kill them? Just edit the map and make a larger docking port then if this is going to be the norm.

Or they can stop buying the station destroying shuttle every round I dunno man.

**Comment by KorPhaeron** [comment]
@Saegrimr 

Happens with literally every new feature

The asteroid shuttle docks fine at this size by the way, the port gets moved into the station when you buy it to cause that crash

**Comment by Saegrimr** [comment]
So why is there a feature to destroy a large portion of the station in the first place? And why is it okay for nonantags to use it but not okay for nonantags to bomb it?

**Comment by ExcessiveUseOfCobblestone** [comment]
that doesn't change the fact you get paid for griefing the crew.

**Comment by HankMoodyExperience** [comment]
**H U G B O X S T A T I O N**

**Comment by SgtxRootbeer** [comment]
@saegrimr
It crashes because of lore reasons...
You know, it's a giant rock with engines strapped to it, it isn't nimble enough to actually get to the station without swapping paint...
(Why it docks perfectly at centcomm, idk. Maybe centcomm has hyperspatial tugboats to assist.)

**Comment by Saegrimr** [comment]
2/3 Headmin confirmation (Third unreachable)

- OK for captains to buy the shuttle (bwoink if done too often, ban if they keep doing it)
- OK  for people who break in and steal the spare to buy the shuttle (bwoink if done too often, ban if they keep doing it)

Reminder that:
- I still haven't banned anybody for this
- I still haven't even warned anybody for this
- I still haven't even told someone to chill out on ordering these
- This entire drama bomb is Goofball starting more bullshit and making a meme PR instead of going with what he first offered which is a shuttle notification.
- Remove Goofball

I'd like to thank everybody involved, myself included, for falling for goof bait when literally nothing happened aside from him trying to remove a bunch of shuttles like a kid kicking his own lego fort down.

**Comment by kevinz000** [comment]
Goddamnit Goofball overexaggerating shit.

**Comment by optimumtact** [comment]
This should be a configurable list for server admins by the way.

**Comment by scoopscoop** [comment]
I really don't mind the new shuttles, it's just that there is no downside for picking a terrible shuttle.

Yeah cargo losing points doesn't count, captains don't give a shit and cargo can't do anything about it.

**Comment by CPTANT** [comment]
Holy mother of salt

Someone *coughSaegrimr* *coughSaltball* should really read rule 10 again.


**10. Losing is part of the game.**
    Your character will frequently die, sometimes without even a possibility of avoiding it. Events will often be out of your control. No matter how good or prepared you are, sometimes you just lose.

**Comment by Arianya** [comment]
"Someone" _cough CPTANT cough Idiotball_ should really read Rule 1


**1. Don't be a dick.**
We're all here to have a good time, supposedly. Going out of your way to seriously negatively impact or end the round for someone with little IC justification is against the rules. Legitimate conflicts where people get upset do happen however, these conflicts should escalate properly and retribution must be proportionate." 

And hey, we even have a spicy precedent to remind people about how specifically that rule is interpreted!

**1.** Random murders are not acceptable nor is the killing of other players for poor or little reasoning such as ‘My character is insane’. Each unjustified kill is normally met with one 24 ban.

**Comment by jughu** [comment]
>im gonna ban people who buy shitty shuttles

>they better gamble the asteroid shuttle and hope that nobody dies

>asteroid shuttle litteraly shows the shuttle area docking thing before it lands

**Comment by CPTANT** [comment]
ohw no the escape shuttle is slightly more dangerous, such a dick move!

**Comment by Arianya** [comment]
Yes, obviously that was my point and not the fact that it can and will 
murder people and thus breaks rule 1.


10/10 for reading comprehension

**Comment by jughu** [comment]
The shuttle litteraly shows where it lands by marking the fucking area with blue circles if people die by this they are either stupid or new guys

**Comment by Bawhoppen** [comment]
close and lock thread for ultimate win

**Comment by Crushtoe** [comment]
@Saegrimr Just want to assure you that it'll slow down. Peopel have already gotten bored of rushing the vault to drain it every round after 3 or 4 days, pretty soon they'll move on past the rock (which doesn't even give the most points.)

Thanks for not banning people while they're just excited to mess around with a shiny new feature and understanding that people have curiosity ❤️

**Comment by factoryman942** [comment]
If it's hard for people to see, just make asteroid shuttle landing zone have a big red overlay like fire alarm for 30 secs before it lands. Or something.

**Comment by nullbear** [comment]
Maybe make them only show if you have < x credits. So you can only get when u need to.

**Pull request closed without merging**